.github/workflows/test.yml

```diff
@@ -77,7 +77,7 @@ jobs:
           fi
 
   compiles:
-    runs-on: buildjet-4vcpu-ubuntu-2204
+    runs-on: depot-ubuntu-22.04-4
```

Comment:
I think this, and the others, should be updated to `depot-ubuntu-24.04`

Reply:
Completed in https://github.com/astriaorg/astria/pull/1047/commits/81a7c4492280530e2d834cce5edb6e1284392472